src/control/server/mgmt_pool.go

```diff
@@ -482,6 +482,14 @@ func (svc *mgmtSvc) poolCreate(parent context.Context, req *mgmtpb.PoolCreateReq
 		return nil, errors.Wrap(err, "unmarshal PoolCreate response")
 	}
 
+	// Zero mem_file_bytes in non-MD-on-SSD mode.
+	if !svc.harness.Instances()[0].GetStorage().BdevRoleMetaConfigured() {
```

Comment:
It would be better to define a helper for this, so that the implementation details aren't copied all over the place.

e.g.
```go
func MdOnSSDEnabled(svc *mgmtSvc) bool {
    if svc == nil || svc.harness == nil || len(svc.harness.Instances()) == 0 {
        return false
    }
    return svc.harness.Instances()[0].GetStorage().BdevRoleMetaConfigured()
}
```

Reply:
Looks like this PR is close to landing, okay if  I this in a follow-up?